src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -243,6 +243,17 @@ class BundleIdArgument(cli.Argument):
             'choices': list(BundleIdPlatform),
         },
     )
+    IDENTIFIER_STRICT_MATCH = cli.ArgumentProperties(
+        key='bundle_id_identifier_strict_match',
+        flags=('--strict-match-identifier',),
+        type=bool,
+        description=(
+            'Only match Bundle IDs that have exactly the same identifier specified by '
+            '`BUNDLE_ID_IDENTIFIER`. By default identifier `com.example.app` also matches '
```

Comment:
what do you think of 
```
f'[`BUNDLE_ID_IDENTIFIER`](#{BUNDLE_ID_IDENTIFIER.key})..'
```
instead of `BUNDLE_ID_IDENTIFIER`?

in this way it will be a nice link (doc.py header anchors is the `key` of an argument)

Reply:
That change would mess up terminal help messages as anchor removal is not supported. If need be we can implement it globally since there are other places too where one option is linked to another.